src/app/collection/pages/nice-challenge/components/curator/curator.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="wrapper">
+  <div class="left-container">
+    <div class="collection-curators">
+      <i class="fa-2x fas fa-users"></i>
+      <h2>COLLECTION CURATORS</h2>
+      <p>
+        Collection curators are responsible for their collection’s 
+        review process and continued quality
+      </p>
+    </div>
+  </div>
+  <div class="curator-container">
+    <div class="curator-card">
+      <img 
+        src="../../../../../../assets/images/collections/Vinnie_Nestler.jpg"
+        alt="Vinnie Nestler, Ph.D."
+      >
+      <p>
+        Vinnie Nestler, Ph.D.
+        Calidornia State University San Berandino
+      </p>
+    </div>
+    <div class="curator-card">
```

Comment:
There is a component in Kyles PR for this component so we can get the curators from the backend.

Reply:
Ok sounds good. I can implement this once his is merged.